q2_alignment/test/test_filter.py

```diff
@@ -82,22 +82,22 @@ def test_gap_boundaries(self):
              skbio.DNA('A', metadata={'id': 'seq2', 'description': ''}),
              skbio.DNA('A', metadata={'id': 'seq3', 'description': ''})]
         )
-        empty_alignment = skbio.TabularMSA(
-            [skbio.DNA('', metadata={'id': 'seq1', 'description': ''}),
-             skbio.DNA('', metadata={'id': 'seq2', 'description': ''}),
-             skbio.DNA('', metadata={'id': 'seq3', 'description': ''})]
-        )
 
         actual = mask(alignment1, max_gap_frequency=1.0, min_conservation=0.0)
         self.assertEqual(actual, alignment1)
 
         actual = mask(alignment2, max_gap_frequency=0.0, min_conservation=0.0)
         self.assertEqual(actual, alignment2)
 
-        eps = np.finfo(float).eps
-        actual = mask(alignment1, max_gap_frequency=1.0-eps,
-                      min_conservation=0.0)
-        self.assertEqual(actual, empty_alignment)
+    def test_error_on_empty_alignment_gap_boundary(self):
+        alignment1 = skbio.TabularMSA(
+            [skbio.DNA('A', metadata={'id': 'seq1', 'description': ''}),
+             skbio.DNA('-', metadata={'id': 'seq2', 'description': ''}),
+             skbio.DNA('-', metadata={'id': 'seq3', 'description': ''})]
+        )
+
+        with self.assertRaises(ValueError):
```

Comment:
Can you use `assertRaisesRegex` since there's some math/formatting going on in the error message? Same comment for the test below.